src/enumTypes.h

```diff
@@ -56,6 +56,76 @@ namespace libsiedler2
         SOUNDTYPE_OTHER = 4
     };
 
+    /**
+     * Enumeration of animals which can be stored in SWD format. 
+     * For each value there's a hex-value assigned for the animal 
+     * representation in SWD.
+     * @see http://settlers2.net/documentation/world-map-file-format-wldswd/
+     */
+    enum ANIMALS
+    {
+        A_None    = 0x00,
+        A_Rabbit  = 0x01,
+        A_Fox     = 0x02,
+        A_Stag    = 0x03,
+        A_Deer    = 0x04,
+        A_Duck    = 0x05,
+        A_Sheep   = 0x06,
+        A_Deer2   = 0x07,
+        A_Duck2   = 0x08,
+        A_Donkey  = 0x09
+    };
+    
+    /**
+     * Enumeration of resource types which can be stored in SWD format.
+     * For each value there's a hex-value assigned for the representation
+     * of the resource in SWD.
+     * @see http://settlers2.net/documentation/world-map-file-format-wldswd/
+     */
+    enum RESOURCES
+    {
+        R_None    = 0x00,
+        R_Water   = 0x21,
+        R_Fish    = 0x87,
+        R_Coal    = 0x40,
+        R_Iron    = 0x48,
+        R_Gold    = 0x50,
+        R_Granite = 0x58
+    };
+    
+    /**
+     * Enumeration for object types stored in SWD. In combination with the
+     * object information it makes a real object (therefore the "info"-comments
+     * in brackets).
+     */
+    enum OBJECT_TYPES
+    {
+        OT_Empty        = 0x00,
+        OT_Stone        = 0x01, // 0x01 - 0x06 (info: 0xCC / 0xCD)
```

Comment:
if there are more of them, use OT_Stone_Begin OT_Stone_End or similar, so one can calculate the range